src/main/java/org/sonar/plugins/ldap/SystemUtilsWrapper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * SonarQube LDAP Plugin
+ * Copyright (C) 2009 SonarSource
+ * dev@sonar.codehaus.org
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonar.plugins.ldap;
+
+import org.apache.commons.lang.SystemUtils;
+
+/**
+ * Wrapper class for accessing SystemUtils APIs
+ */
+public class SystemUtilsWrapper {
```

Comment:
This class could be replaced by `org.sonar.api.utils.System2` which is provided by API (see method `isOsWindows()`). Like SystemUtilsWrapper it can be mocked too.